Document some default parameters

corinwagen opened the pull request:
Cf. #19 -- took a first pass at just writing down the most useful parameters and what their values are. Not sure if this is the best place for the file; it seemed like too much for the root ``README.md``.

Criticism/clarification is welcomed!

kjelljorner commented on docs/README.md:
```diff
@@ -0,0 +1,80 @@
+# docs
```
I would also list:
- `imultistate` (another one of the specialities of DL-FIND)

kjelljorner reviewed (commented):
Overall looks good but some minor things to correct/add. Seems like they changed the GitHub windows runners so I need to update the wheel building, but you can go ahead to make the revisions in the meantime.

kjelljorner commented on docs/README.md:
```diff
@@ -0,0 +1,80 @@
+- 4: verbose
+- 6: debug
+
+#### ``icoord``
```
Also, please create a link from the top README

corinwagen commented:
I think I've addressed the comments. I also added documentation for ``imicroiteration``, since that's the only one from your original list that wasn't there.

kjelljorner commented:
Fantastic, now I just have to butt my head against the GitHub Actions wall for a while...

corinwagen commented:
Have fun :/
